libs/wingc/src/capture.rs

```diff
@@ -178,40 +176,15 @@ fn scan_for_inflights_in_arglist(args: &ArgList, diagnostics: &mut Diagnostics)
 }
 
 fn scan_captures_in_call(
-	reference: &Reference,
+	callee: &Expr,
 	args: &ArgList,
 	env: &SymbolEnv,
 	statement_idx: usize,
 	diagnostics: &mut Diagnostics,
 ) -> Vec<Capture> {
 	let mut res = vec![];
-	if let Reference::NestedIdentifier { object, property } = reference {
-		res.extend(scan_captures_in_expression(&object, env, statement_idx, diagnostics));
-
-		// If the expression evaluates to a resource we should check what method of the resource we're accessing
```

Comment:
I see you got rid of this. I guess it makes sense because it's not relevant to capturing, it's more of a type-checking thing and doesn't affect the output.